cdap-data-fabric/src/main/java/co/cask/cdap/data2/datafabric/dataset/service/executor/DatasetAdminService.java

```diff
@@ -234,6 +234,8 @@ public DatasetAdmin call() throws Exception {
         });
       } catch (Exception e) {
         Throwables.propagateIfInstanceOf(e, IOException.class);
+        Throwables.propagateIfInstanceOf(e, DatasetManagementException.class);
```

Comment:
in the callable, the `datasetInstantiator.getDatasetAdmin` throws the DatasetManagementException. So we shouldn't wrap it.